.golangci.yml

```diff
@@ -10,7 +10,7 @@ run:
   issues-exit-code: 1
   tests: true
   build-tags: [""]
-  skip-dirs:  [""]
+  skip-dirs:  ["proto"]
```

Comment:
this should be /protos

This is why your code coverage is broken.

Reply:
I'll fix

Reply:
Also are .pb.go files being ignored?

Reply:
I believe golangci ignores auto generated files..

as discussed, I will remove `skip-dirs:  ["proto"]` since golangci ignores non go files anyway.